presentation-example/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "airbnb-base",
```

Comment:
I added the airbnb eslint rule set to both projects to clean up some of the code. Please let me know if that is not desired.